diff --git a/dist/resources/helpers/signature-title.js b/dist/resources/helpers/signature-title.js
index 38ade0bf586ad0c1bea30140b263c7e359cb5d1d..3128b2db6edc858ded47070b51194195aefa4265 100644
--- a/dist/resources/helpers/signature-title.js
+++ b/dist/resources/helpers/signature-title.js
@@ -13,11 +13,13 @@ function default_1(theme) {
         if (this.parent && ((_a = this.parent.flags) === null || _a === void 0 ? void 0 : _a.length) > 0) {
             md.push(this.parent.flags.map((flag) => `\`${flag}\``).join(' ') + ' ');
         }
+        // these patch changes can be reverted if this PR gets published
+        // https://github.com/tgreyuk/typedoc-plugin-markdown/pull/401
         if (accessor) {
-            md.push(`\`${accessor}\` **${this.name}**`);
+            md.push(`\`${accessor}\` **${utils_1.escapeChars(this.name)}**`);
         }
         else if (this.name !== '__call' && this.name !== '__type') {
-            md.push(`**${this.name}**`);
+            md.push(`**${utils_1.escapeChars(this.name)}**`);
         }
         if (this.typeParameters) {
             md.push(`<${this.typeParameters